app/views/telephone_appointments/ineligible.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="l-grid-row">
+  <div class="l-column-two-thirds">
+    <h1>Sorry, you’re unable to have an appointment</h1>
+
+    <p>You must be aged <strong>50 or over</strong> and have a
+    <strong>defined contribution</strong> pension.</p>
+
+    <div role="note" aria-label="Information" class="application-notice info-notice">
+      <p>Pension Wise only provides guidance on what you can do with a defined contribution pension. For guidance on a defined benefit pension go to <a rel="external" href="http://www.pensionsadvisoryservice.org.uk/">The Pensions Advisory Service</a>.</p>
+    </div>
+
+    <p>Find out more if you’re not sure what <a
+      href="/pension-types">type of pension</a> you have.</p>
```

Comment:
Is this a line break deliberado? Compared to the longun above it feels a bit out.